BUILDING.md

```diff
@@ -417,41 +417,51 @@ To build the documentation:
 
 This will build Node.js first (if necessary) and then use it to build the docs:
 
-```console
-$ make doc
+```bash
+make doc
 ```
 
 If you have an existing Node.js build, you can build just the docs with:
 
-```console
-$ NODE=/path/to/node make doc-only
+```bash
+NODE=/path/to/node make doc-only
 ```
 
-To read the documentation:
+To read the man page:
 
-```console
-$ man doc/node.1
+```bash
+man doc/node.1
 ```
 
-If you prefer to read the documentation in a browser,
-run the following after `make doc` is finished:
+If you prefer to read the full documentation in a browser, run the following.
 
-```console
-$ make docopen
+```bash
+make docserve
 ```
 
-This will open a browser with the documentation.
+This will build the docs, spin up a static file server, and provide a URL to
+where you may browse the documentation locally.
+
+If you're comfortable viewing the documentation using the program your operating
+system has associated with the default web browser, run the following.
```

Comment:
```suggestion
system has associated with the default web browser, run the following:
```

Reply:
College textbooks do it like how I had it.

Reply:
OK but that's inconsistant with the rest of the document…

Reply:
This is indeed consistent. It's a full sentence that stands on its own. The others do not stand on their own.